modules/cluster/variables.tf

```
@@ -27,12 +27,14 @@ variable "cluster_configuration" {
 }
 
 variable "cluster_settings" {
-  description = "Configuration block(s) with cluster settings. For example, this can be used to enable CloudWatch Container Insights for a cluster"
-  type        = map(string)
-  default = {
-    name  = "containerInsights"
-    value = "enabled"
-  }
+  description = "List of configuration block(s) with cluster settings. For example, this can be used to enable CloudWatch Container Insights for a cluster"
+  type        = list(map(string))
```

```suggestion
  type        = any
```

`any` is more broad and not introducing BC break.